src/main/java/com/gtnewhorizons/gtnhgradle/modules/ToolchainModule.java

```diff
@@ -97,6 +99,18 @@ public void apply(GTNHGradlePlugin.@NotNull GTNHExtension gtnh, @NotNull Project
                 c.includeGroup("org.lwjgl");
             });
         });
+        repos.maven(mvn -> {
+            mvn.setName("Cleanroom Maven");
+            mvn.setUrl("https://maven.cleanroommc.com");
+        });
+        repos.maven(mvn -> {
+            mvn.setName("GTCEu Maven");
+            mvn.setUrl("https://maven.gtceu.com");
+        });
+        repos.maven(mvn -> {
+            mvn.setName("BlameJared Maven");
+            mvn.setUrl("https://maven.blamejared.com/");
+        });
```

Comment:
You should be able to safely remove this, as lwjgl3ify 1.12 is on gtceu maven, and mixinbooter+deps are on cleanroom maven

Reply:
JEI is on this maven

Reply:
These mavens should also just be enabled for 1.12, each additional maven adds additional risk for the plugin breaking builds if any of them go offline. Gradle fails the build if it tries a maven and the maven errors out, regardless if the artifact is present on other mavens.